template_config.yml

```diff
@@ -85,11 +86,13 @@ tasking_allow_async_unsafe: true
 test_azure: true
 test_bindings: false
 test_cli: false
+test_deprecations: true
```

Comment:
PR fails because we are testing pulpcore deprecations

Reply:
@fao89 should we keep testing it? or change to false in this PR?

Reply:
I don't know, it is up to galaxy
I personally think this deprecation test could be moved to your nightly latest